scipy/ndimage/tests/test_filters.py

```diff
@@ -922,6 +922,7 @@ def test_filter_axes(self, filter_func, extra_args, size0, size, axes, xp):
         expected = filter_func(array, *extra_args, all_sizes)
         xp_assert_close(output, expected)
 
+    @skip_xp_backends("cupy", reasons=["gh-21486"])
```

Comment:
This is https://github.com/cupy/cupy/pull/8339, like in the above test, right? Maybe better to reference that for the skip reason.

Reply:
ok, I think I've revised accordingly now